src/main/java/bio/terra/cli/command/resource/create/GcpNotebook.java

```diff
@@ -47,8 +39,7 @@
               + "The instance name must be 1 to 63 characters long and contain only lowercase "
               + "letters, numeric characters, and dashes. The first character must be a lowercase "
               + "letter and the last character cannot be a dash. If not specified, an "
-              + "auto-generated name based on your email address and time will be used.",
-      defaultValue = AUTO_GENERATE_NAME)
+              + "auto-generated name based on your email address and time will be used.")
```

Comment:
Is this true? you earlier comment said that wsm generates an instance id like terra-uuid-notebook.

Also, is the solutions team ok with this change in behavior? Will wrote this logic and I don't know whether he did that based on input from them or not.

Reply:
i'm copying this logic to WSM side in a separate PR.

Reply:
ah ok. are you planning to merge the wsm PR before this one then? or I guess you could move the notebook part of this to a separate PR and just merge the other stuff until the wsm change is ready.

Reply:
https://github.com/DataBiosphere/terra-workspace-manager/pull/525 I merged the WSM change. I will update the wsm version of this PR in the gradle.